src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Shape {
+    private double height;
+    private double width;
+
+    public Rectangle(String color, double height, double width) {
+        super(Colors.valueOf(color));
+        this.height = Math.round(Math.abs(height));
+        this.width = Math.round(Math.abs(width));
+
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    public double getWidth() {
+        return width;
+    }
+
+    public void setWidth(double width) {
+        this.width = width;
+    }
+
+    @Override
+    public double calculateArea() {
+        return height * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Shape: square, area: " + calculateArea()
```

Comment:
The `draw` method refers to the shape as a 'square'. Consider changing this to 'rectangle' to avoid confusion, as a rectangle does not necessarily have equal height and width.